assets/css/style-mobile.css

```diff
@@ -444,6 +446,44 @@ h4 {
   margin: 20px 0;
 }
 
+.web-nav {
+  display: none;
+  background: rgba(96, 112, 255, 0.88);
+  height: 100%;
+  width: 100%;
+  top: 0;
+  margin-left: -15px;
+  position: fixed;
+}
+
+.web-nav img {
+  margin-left: 85%;
+  margin-top: 10%;
+}
```

Comment:
- [ ] Nice work implementing the closing button :clap: However It would be nice to reduce the size of the button a little bit to follow the Figma design.
### Your design :point_down: 
![Screenshot from 2023-06-21 21-49-13](https://github.com/kambereBr/Portfolio/assets/91142432/cae38a1d-849d-4de1-b351-31cdae03eff1)
### Figma design :point_down: 
![Screenshot from 2023-06-21 21-49-58](https://github.com/kambereBr/Portfolio/assets/91142432/7cc7beab-2a1d-4f89-9c20-7afdb0153d26)